google/async_resumable_media/requests/_request_helpers.py

```diff
@@ -127,10 +131,19 @@ def http_request(
     Returns:
         ~requests.Response: The return value of ``transport.request()``.
     """
+
+    # NOTE(asyncio/aiohttp): Sync versions use a tuple for two timeouts,
+    # default connect timeout and read timeout. Since async requests only
+    # accepts a single value, this is using the connect timeout. This logic
+    # diverges from the sync implementation.
```

Comment:
Can you clarify what the meaning of this timeout is? Would we run into timeout issues if a blob is too large for example?

Reply:
The way aiohttp and requests do timeouts isn't identical, so there are some differences.

That said, digging into this, we may be able to continue to have separate connect and between-read timeouts. https://docs.aiohttp.org/en/stable/client_quickstart.html#timeouts